docs/toc.yml

```diff
@@ -768,6 +776,12 @@ items:
     - name: Autogenerate JSON metadata
       href: excel/custom-functions-json-autogeneration.md
       displayName: Excel, Custom Functions
+    - name: Data types and custom functions overview (preview)
```

Comment:
Recommend: Shorten these titles in the TOC. Just say "Data types overview" since "Custom Functions" context comes from the parent tab.

Reply:
I'm concerned that someone might see "Data types overview" here and think it's the main data types article. What do you think? Since we have data types content in two nodes — Excel JS API data types content, and the custom functions integration data types content.

Reply:
ok